src/pages/workspace/accounting/xero/advanced/XeroAdvancedPage.tsx

```diff
@@ -19,7 +21,14 @@ function XeroAdvancedPage({policy}: WithPolicyConnectionsProps) {
     const policyID = policy?.id ?? '';
     const xeroConfig = policy?.connections?.xero?.config;
     const {autoSync, pendingFields, sync} = xeroConfig ?? {};
-    const xeroData = policy?.connections?.xero?.data;
+    const {bankAccounts} = policy?.connections?.xero?.data ?? {};
+
+    const selectedBankAccountName = useMemo(() => {
+        const {invoiceCollectionsAccountID} = sync ?? {};
```

Comment:
Does this have to be inside `useMemo`? Can we take this outside and add `invoiceCollectionsAccountID` as dependency?

Reply:
updated!